owners/OWNERS.example

```diff
@@ -132,10 +133,15 @@
 
   // The root-level owners file--and ONLY the root-level owners file--may
   // specify an optional `reviewerPool` property, providing a list of GitHub
-  // users/teams. If present, the owners check will require that at least one
-  // member of this team has given approval. It is possible that one reviewer
-  // is both a member of this reviewer team and provides owners coverage.
-  reviewerPool: ["ampproject/reviewers-amphtml", "approver-bot"]
+  // teams/users/bots. If present, the owners check will require that at least
+  // one member of this pool has given approval. It is possible that a given
+  // reviewer is both a member of this reviewer pool and provides owners
+  // coverage.
+  reviewerPool: [
+    "ampproject/reviewers-amphtml",
+    "user-approver",
+    "bot-approver[bot]",
```

Comment:
Let's update the result Reviewers list below too (line 149ish)

Reply:
Good catch. Done.